code/modules/clothing/head/misc.dm

```diff
@@ -135,7 +135,7 @@
 		return
 	if(slot == SLOT_HEAD)
 		user.grant_language(/datum/language/piratespeak/)
-		to_chat(user, "You suddenly know how to speak like a pirate!")
+		to_chat(user, "<span class='boldnotice'>You suddenly know how to speak like a pirate!</span>")

```

Comment:
Feels like this should just be a normal notice. Fine as it is though.

Reply:
I was thinking that notifications like these, which will likely happen only once in a round, could be emphasized a bit more. I'll have a look at things like these later.

Reply:
Thing is that this happens every time you put on the hat and/or remove it.